maquetado/scss/styles.scss

```diff
@@ -50,6 +50,32 @@
   margin-right: 100px;
 }
 
+.books-list {
+  align-content: center;
+  background-color: $wild-sand;
```

Comment:
Like the other PR, I would set this inside a body class. Like this you have a white space left on both sides (Right & Left)